pennylane/qchem/observable_hf.py

```diff
@@ -76,6 +80,25 @@ def fermionic_observable(constant, one=None, two=None, cutoff=1.0e-12):
     if indices_sort:
         indices_sort = qml.math.array(indices_sort)
 
+    if fs:
+        sentence = FermiSentence({FermiWord({}): constant[0]})
+        for c, o in zip(coeffs[indices_sort], sorted(operators)):
+            if len(o) == 2:
+                sentence.update({FermiWord({(0, o[0]): "+", (1, o[1]): "-"}): c})
+            if len(o) == 4:
+                sentence.update(
+                    {FermiWord({(0, o[0]): "+", (1, o[1]): "+", (2, o[2]): "-", (3, o[3]): "-"}): c}
+                )
+        sentence.simplify()
+
+        return sentence
+
+    warnings.warn(
+        "This function will return a fermionic operator by default in the next release. For details,"
+        " see the Fermionic Operators tutorial:"
+        " https://pennylane.ai/qml/demos/tutorial_fermionic_operators"
+    )
```

Comment:
Again, I would consider mentioning how to switch to the upcoming default already

Reply:
Added both.